### ISSUE TITLE
Throws error when no permission to read file

### ISSUE THREAD

**Issue by benogle** [atom/season#5]
* `chmod 000 somefile.cson`
* Try to read in `somefile.cson`

![screen shot 2015-01-13 at 12 06 34 pm](https://cloud.githubusercontent.com/assets/69169/5728137/e2cc0dc8-9b1c-11e4-978c-1e9217ac6f9a.png)

Ideally, I would get an ENOENT error back in the callback.

**Comment by benogle** [comment]
Trace in text form

```
TypeError: Cannot read property 'trim' of undefined
  at /Users/ben/github/snippets/node_modules/season/lib/cson.js:116:23
  at fs.js:224:20
  at Object.oncomplete (fs.js:93:15)
```

**Comment by kevinsawicki** [comment]
Thanks, fixed and published in 5.0.4
